modules/nodes-nodes-audit-log-basic.adoc

```diff
@@ -6,27 +6,12 @@
 = About the API Audit Log
 
 Audit works at the API server level, logging all requests coming to the server.
-Each audit log contains two entries:
 
-. The request line containing:
-.. A Unique ID allowing to match the response line (see #2)
-.. The source IP of the request
-.. The HTTP method being invoked
-.. The original user invoking the operation
-.. The impersonated user for the operation (`self` meaning himself)
-.. The impersonated group for the operation (`lookup` meaning user's group)
-.. The namespace of the request or <none>
-.. The URI as requested
+The audit policy object structure is defined in the `audit.k8s.io` API group.
```

Comment:
What does this mean? How/where would a user actually go to find the structure? I don't see this in our API docs at all.

Reply:
Yeah, I couldn't find those either, I'm not sure how we generate API docs these days but we're missing this bit.

Reply:
https://github.com/kubernetes/kubernetes/blob/ea3eeb494a528edcc02f8887d3098f33fca547b3/staging/src/k8s.io/apiserver/pkg/apis/audit/v1/types.go#L72-L136 is the structure that describes this. Not sure how we could expose that best.